WePay/Models/WPConfig.m

```diff
@@ -55,7 +64,7 @@ - (NSDictionary *) toDict
     NSMutableDictionary *dict = [NSMutableDictionary dictionary];
     
     [dict setValue:self.clientId ? self.clientId : [NSNull null] forKey:@"clientId"];
-    [dict setValue:self.environment ? self.environment : [NSNull null] forKey:@"environment"];
+    [dict setValue:@(self.environment) forKey:@"environment"];
```

Comment:
I wasn't sure about where this dictionary is used, should the value for the `environment` key be something specific?